legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -91,6 +91,19 @@
                <div class="pt-2">{{filing_date_time}}</div>
                <div class="pt-2">{{report_date_time}}</div>
             </td>
+         {% elif header.reportType == 'correction' %}
```

Comment:
Will impact all corrections, did you try this with any other kinds of corrections?